### PR TITLE
re-org addons

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR introduces a minor change to re-org addons into its own collapsible sidebar.

### CODE REVIEW

File: docs/advanced/addons/_category_.json
Comment (LucasSaintarbor) [inline docs/advanced/addons/_category_.json:3]:
```suggestion
    "label": "Available Add-ons",
```

File: docs/advanced/addons/rancher-vcluster.md
Comment (LucasSaintarbor) [inline docs/advanced/addons/rancher-vcluster.md:13]:
```suggestion
The `rancher-vcluster` addon allows you to run Rancher Manager as a workload on the underlying Harvester cluster and is implemented using [vcluster](https://www.vcluster.com/).
```

**Comment by ibrokethecloud** [comment]
@LucasSaintarbor once this is merged, i can also port the sidebar setup for v1.1 as well.

**Comment by chrisho** [comment]
Please rebase the upstream to fix the ci error.

File: docs/advanced/addons/rancher-vcluster.md
Comment (vickyhella) [inline docs/advanced/addons/rancher-vcluster.md:13]:
```suggestion
The `rancher-vcluster` add-on allows you to run Rancher Manager as a workload on the underlying Harvester cluster and is implemented using [vcluster](https://www.vcluster.com/).
```

File: docs/advanced/addons/seeder.md
Comment (vickyhella) [inline docs/advanced/addons/seeder.md:13]:
```suggestion
The `harvester-seeder` add-on lets you perform out-of-band operations on underlying nodes. 
```

**Review by vickyhella — COMMENTED** [review]
LGTM, but if we are changing `addon` to `add-on`, better globally change the terms for consistency

File: docs/advanced/addons/seeder.md
Comment (vickyhella) [inline docs/advanced/addons/seeder.md:8]:
```suggestion
  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/advanced/seeder"/>
```

File: docs/advanced/addons.md
Comment (ibrokethecloud) [reply docs/advanced/addons.md:19]: i will rename it back addons

File: docs/advanced/addons/rancher-vcluster.md
Comment (ibrokethecloud) [reply docs/advanced/addons/rancher-vcluster.md:13]: we will stick with `addon` for now since UI is already showing `Addon`

File: docs/advanced/addons/seeder.md
Comment (ibrokethecloud) [reply docs/advanced/addons/seeder.md:13]: we will stick with `addon` for now since UI is already showing `Addon`

**Review by guangbochen — APPROVED** [review]
LGTM, @ibrokethecloud can u please help to rebase and squash those commits? thanks.

**Comment by vickyhella** [comment]
Hit the same issue of #415. Need #416 to be merged first to pass the CI
CC @guangbochen

**Review by vickyhella — APPROVED** [review]

**Pull request merged**
